Sources/MapboxNavigation/PassiveLocationProvider.swift

```diff
@@ -76,11 +76,11 @@ open class PassiveLocationProvider: NSObject, LocationProvider {
     }
 
     public func startUpdatingLocation() {
-        locationManager.startUpdatingLocation()
+        locationManager.startTripSession()
     }
 
     public func stopUpdatingLocation() {
-        locationManager.systemLocationManager.stopUpdatingLocation()
+        locationManager.stopTripSession()
     }
```

Comment:
I appreciate that renaming these methods clarifies the lifespan of a billing session, but losing the consistency between PassiveLocationManager method names versus PassiveLocationProvider and CLLocationManager method names would be an unfortunate tradeoff. A developer is already used to thinking in terms of starting and stopping location updates in a location manager, since that’s the terminology that Core Location uses. A trip-based billing session beginning and ending would be only one of multiple consequences of starting and stopping location updates.

The map SDK automatically calls `stopUpdatingLocation()` and `startUpdatingLocation()` when the user sends the application to the background and foreground, respectively. Is it clear to the developer that using the application switcher or lock screen always resets the trip session, as these lines imply? It’s surprising that these methods seems like they do something entirely different than `stopUpdatingHeading()` and `startUpdatingHeading()`, which get called at the same time.

Reply:
Thank you for feedback. After reading this multiple times including [this](https://github.com/mapbox/mapbox-navigation-ios/pull/3362#discussion_r709622432) comment I'm inclining to:
- I revert all the changes I've made to API surface level and just keep the code that doesn't send location updates to Navigator when trip session is paused. This will give total control to devs except that they can't control when trip session first starts.

Reply:
Done. The PR simplified to the bare minimum.